artifactory/commands/utils/configfile.go

```diff
@@ -125,6 +129,31 @@ func goConfigeFile(configResult *ConfigFile) error {
 	return npmConfigeFile(configResult)
 }
 
+func readGradleGlobalConfig(configResult *ConfigFile) error {
```

Comment:
The code in this function seems to be duplicated in the code.
I suggest we stop supporting the old *mvnc* and *gradlec* commands format. Since these commands are interactive (can be run by humans only), we can safely stop supporting them.

Reply:
Removed.